core/src/main/kotlin/transformers/pages/DefaultPageMergerStrategy.kt

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.dokka.transformers.pages
+
+import org.jetbrains.dokka.pages.*
+
+object DefaultPageMergerStrategy : PageMergerStrategy() {
```

Comment:
The idea of this class was that it "merges" pages by just returning the first one and reporting warning if group contains more than one page.

Reply:
Now there are Default and SamePage strategies.